### ISSUE TITLE
[Mobile Android] Feed | Comment | Send button placement is off (when phone display size is set to large-largest)

### ISSUE THREAD

**Issue by hellynyc** [FightPandemics/FightPandemics#1054]
Device/OS/Browser: OnePlus 7 Pro / OS10 / Chrome

Expected Result: The "Send" button should be on the right of the text field.

Actual Result: When the phone's display setting is set to large/larger/largest, the "Send" button moves to the bottom of the page.
![Screenshot_20200707-012015](https://user-images.githubusercontent.com/52841932/86720112-b3567500-bff2-11ea-8d83-adecd7e47f72.jpg)

**Comment by hellynyc** [comment]
Related to issue #1021
